app/helpers/config.js

```diff
@@ -0,0 +1,5 @@
+export const public_data =
```

Comment:
use CONST_CASE for consts while we're here.

move to `constants/authorities.js`

Reply:
Sure, thanks